app/src/main/java/io/homeassistant/companion/android/database/widget/StaticWidgetEntity.kt

```diff
@@ -13,5 +13,9 @@ data class StaticWidgetEntity(
     @ColumnInfo(name = "attribute_id")
     val attributeId: String?,
     @ColumnInfo(name = "label")
-    val label: String?
+    val label: String?,
+    @ColumnInfo(name = "text_size")
+    val textSize: Int?,
+    @ColumnInfo(name = "separator")
+    val separator: String?
```

Comment:
Based on your other code it looks like these are required and always have a value, why have them nullable?

Reply:
Makes sense. I hadn't fully understood what question mark in Kotlin means. Fixed now.